docs/usage/import-automated.md

```diff
@@ -84,11 +84,11 @@ Additional configuration options:
 Ensuring a secure and well-organized environment is paramount in OpenCTI. Here are two recommended best practices to enhance security, traceability, and overall organizational clarity:
 
 1. Create a dedicated user for each source: Generate a user specifically for feed import, following the convention `[F] Source name` for clear identification. Assign the user to the "Connectors" group to streamline user management and permission related to data creation. Please [see here](../deployment/connectors.md#connector-token-section) for more information on this good practice.
-2. Establish a dedicated Organization for the source: Create an organization named after the data source for clear identification. Assign the newly created organization to the "Default author" field in feed import configuration if available.
+2. Assign the users to a specific group, in charge of creating knowledge in your platform and assign to this group a max confidence level. This will ensure that all your users will be able to create and update knowledge in your platform with a confidence level that match your needs. **Without assigning a max confidence level, your users won’t be able to create or update any data.** See in this [page](https://docs.opencti.io/latest/administration/users/) how to assign a confidence level.
```

Comment:
I don't think we should say that here. Generally speaking, to create a user account, you need to refer to the appropriate documentation, which must contain this information.

Reply:
Yes, I agree, I'll remove this sentence.